### PR TITLE
Html code generation tests: Chart layout

### PR DESCRIPTION
**Pull request by WhiteBlackGoose**
Work in progress 🦆 . Feel free to correct me if I got into a wrong direction.

I will revert Sln and playground files at the moment when the PR will be ready

### CODE REVIEW

File: tests/Plotly.NET.Tests/HtmlCodegen/ChartLayout.fs [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        |> Chart.withX_AxisStyle ("X axis title quack quack", MinMax = (-1.,10.))
+        |> Chart.withY_AxisStyle ("Y axis title boo foo", MinMax = (-1.,10.))
+    plot1
+
+
+[<Tests>]
+let ``Axis styling tests`` =
+    testList "Axis styling tests" [
+        testCase "X With axis has title" ( fun c ->
+            "X axis title quack quack"
```
Comment (kMutagene) [inline tests/Plotly.NET.Tests/HtmlCodegen/ChartLayout.fs]: I think you can be a little stricter in all of these cases. Something like this: `"\"title\":\"X axis title quack quack\""`

**Comment by WhiteBlackGoose** [comment]
Oops now I messed the history up. I will fix it later

File: tests/Plotly.NET.Tests/HtmlCodegen/ChartLayout.fs [hunk 2]
```diff
@@ -0,0 +1,296 @@
...
+    twoXAxes1
+
+[<Tests>]
+let ``Multiple Axes styling`` =
+    testList "Multiple Axes styling tests" [
+        testCase "Layout" ( fun () ->
+            "var layout = {\"yaxis\":{\"title\":\"axis 1\",\"side\":\"left\"},\"yaxis2\":{\"title\":\"axis2\",\"side\":\"right\",\"overlaying\":\"y\"}};"
+            |> chartGeneratedContains multipleAxesChart
+        );
+        testCase "Passing args to the function" ( fun () ->
```
Comment (kMutagene) [inline tests/Plotly.NET.Tests/HtmlCodegen/ChartLayout.fs]: I think this test belongs better into the `Html layout tests`

Comment (WhiteBlackGoose) [reply tests/Plotly.NET.Tests/HtmlCodegen/ChartLayout.fs]: Indeed

**Pull request merged**